Neos.Flow/Classes/Mvc/Routing/Dto/Action.php

```diff
@@ -0,0 +1,94 @@
+<?php
+declare(strict_types=1);
+namespace Neos\Flow\Mvc\Routing\Dto;
+
+/*
+ * This file is part of the Neos.Flow package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Flow\Annotations as Flow;
+use Neos\Flow\Mvc\ActionRequest;
+
+/**
+ * @Flow\Proxy(false)
+ */
+final class Action
```

Comment:
As discussed: We should rename this (maybe to `ActionUriSpecification`?  and add a property for QueryParameters and rename addtionalArguments to `routingArguments`

Reply:
done ;) (thanks for the note ^^)